### PR TITLE
Fix/running tests

### PR DESCRIPTION
**Pull request by GalaxiasKyklos**
#### What does this PR do?
Fixed the tests that did not run or did not pass
- Modified the tests script so no user interaction is needed to run all tests
- Added a npm dependency for App.test.js

#### Where should the reviewer start?
This only fixes the crashing tests, it does not do anything with the functionality.

### CODE REVIEW

**Comment by alan-maldonado** [comment]
LGTM,

I think we should only use eslint with the src folder and only for js files not for testing.

what do you think @codinronan ?

"lint": "./node_modules/.bin/eslint **/*.js;" ->
"lint": "eslint src/ --ignore-pattern *.test.js"

File: src/components/AdminAnswers.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { renderQuestionRow } from './ListQuestions';
+import { renderAnswerModal } from './ListQuestions';
+import List from './ListQuestions';
+import { shallow } from 'enzyme';
+
+it('Question row renders without crashing', () => {
```
Comment (alan-maldonado) [inline src/components/AdminAnswers.test.js]: can we wrap this test file with describe? Just as App component

File: src/components/Intro.test.js [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
-import Intro from './Intro';
+import Choice from './Intro';
+import { mount } from 'enzyme';
+import storeFake from '../stores';
+import { Provider } from 'react-redux';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<Intro />, div);
+describe('testing intro', () => {
```
Comment (alan-maldonado) [inline src/components/Intro.test.js]: Just put the name of the component ^

Comment (GalaxiasKyklos) [inline src/components/Intro.test.js]: without testing?

File: src/components/AdminAnswers.test.js (hunk 1 above)
Comment (GalaxiasKyklos) [inline src/components/AdminAnswers.test.js]: sure

**Comment by codinronan** [comment]
@GalaxiasKyklos @alan-maldonado I'd rather lint tests, too. The point of linting is to maintain consistent standards to make maintenance, etc. easier, and that applies to tests, too. :)

File: scripts/test.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const argv = process.argv.slice(2);
 
 // Watch unless on CI or in coverage mode
 if (!process.env.CI && argv.indexOf('--coverage') < 0) {
-  argv.push('--watch');
+  argv.push('--watchAll --forceExit');
```
Comment (rominavarela) [inline scripts/test.js]: Why is this change needed? More of being curious

File: src/components/AdminAnswers.test.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    const div = document.createElement('renderQuestionRow');
+    ReactDOM.render(<renderQuestionRow />, div);
+  });
+
+  it('Answer modal renders without crashing', () => {
+    const div = document.createElement('renderAnswerModeal');
+    ReactDOM.render(<renderAnswerModal />, div);
+  });
+
+  it('get length of answer as Admin', () => {
```
Comment (rominavarela) [inline src/components/AdminAnswers.test.js]: Awesome! looks very clean and clear

File: src/components/Intro.test.js (hunk 2 above)
Comment (alan-maldonado) [inline src/components/Intro.test.js]: yeap

File: scripts/test.js (hunk 3 above)
Comment (GalaxiasKyklos) [inline scripts/test.js]: --watch ran the only after you pressed the a key; --watchAll runs them all and --forceExist ends the tests session after running them (without having to press the q key)

File: src/components/ListQuestions.js [hunk 5]
```diff
@@ -170,7 +172,7 @@ class List extends Component {
         <div className="question">
           {question.question}
           {this.displayEdit(question.can_edit && !question.answer)}
-          {!question.answer &&
+          {!question.answer && checkForAdminStatus() &&
             <a onClick={() => this.openAnswerModal(question.question_id)}> Answer</a>}
```
Comment (rominavarela) [inline src/components/ListQuestions.js]: please add a line break after and before the block braces `{`

File: src/components/QuestionInput.test.js [hunk 6]
```diff
@@ -1,8 +1,25 @@
 import React from 'react';
```
Comment (rominavarela) [inline src/components/QuestionInput.test.js]: If you find time to add more complexity for this test:
- fake the postQuestion function
- verify posting empty, anonymous and authenticated data
